bundles/org.openhab.binding.solarmax/src/main/resources/OH-INF/i18n/solarmax.properties

```diff
@@ -12,8 +12,10 @@ thing-type.solarmax.inverter.description = Basic thing for the SolarMax Power In
 
 thing-type.config.solarmax.inverter.host.label = Host
 thing-type.config.solarmax.inverter.host.description = Hostname or IP Address
-thing-type.config.solarmax.inverter.portNumber.label = Port
+thing-type.config.solarmax.inverter.portNumber.label = Port Number
 thing-type.config.solarmax.inverter.portNumber.description = Port Number (defaults to 12345)
+thing-type.config.solarmax.inverter.deviceAddress.label = Device Address
+thing-type.config.solarmax.inverter.deviceAddress.description = Device Address (defaults to 1)
```

Comment:
Please regenerate, text has now changed to: "Device address for devices connected serially (defaults to 1)"

Reply:
Ran spotless:apply, committed and pushed.

Reply:
You also need to regenerate this file, see https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file